src/plugins/kibana_react/public/core/mock.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Core } from './types';
+
+const createSetupContractMock = () => {
```

Comment:
This file is only for testing right?

Reply:
Yes, these are Jest mocks only for testing.

In the next PR, I want to rename this file to `mock.test.ts` and try to require the mocks from Core (had some problems doing that).

Reply:
> Would be curious to hear about the issues you were running into.

@lukeelmers tests on CI fail, hopefully I just need to rename to `*.test.ts` extension no make them pass. Will try next PR.